beacon-chain/rpc/validator/exit.go

```diff
@@ -0,0 +1,75 @@
+package validator
```

Comment:
Not sure if we should have another `validator` server, this creates confusion with `proposer`, `attester` and `aggregator`

I think it's perfectly fine to put this under `proposer`. Proposer is ultimately the one  packages `exit` object in the block. And if that's not well known, we can improve upon docs

Reply:
Hey @terencechain and @mcdee, after thinking about this more - we will overhaul the RPC package to ensure we implement Ethereum APIs, so placing this exit func in the validator/ subpackage is correct

Reply:
I see the trade offs. Let's hold off on this PR until we have refactored our validator RPC package then. @rauljordan and I will work on this today